bindings/pyroot/pythonizations/python/ROOT/_pythonization/_roofit/_roodatahist.py

```diff
@@ -50,3 +50,190 @@ def plotOn(self, *args, **kwargs):
         # The keywords must correspond to the CmdArg of the `plotOn` function.
         args, kwargs = _kwargs_to_roocmdargs(*args, **kwargs)
         return self._plotOn(*args, **kwargs)
+
+    @property
+    def shape(self):
+        import ROOT
```

Comment:
We can remove this import as it is not used?

Reply:
Thanks! You're right.